dispatch-service/src/main/java/de/muenchen/oss/swim/dispatcher/application/usecase/DispatcherUserCase.java

```diff
@@ -207,7 +215,9 @@ protected void processProtocolFile(final UseCase useCase, final File file) {
             // tag protocol as processed
             fileSystemOutPort.tagFile(file.bucket(), file.path(), Map.of(
                     swimDispatcherProperties.getProtocolStateTagKey(), swimDispatcherProperties.getProtocolProcessedStateTageValue()));
-
+            // move protocol
+            final String destPath = useCase.getFinishedPath(swimDispatcherProperties, file.path());
+            fileSystemOutPort.moveFile(file.bucket(), file.path(), destPath);
```

Comment:
Here would be the question if we should also move it if a mismatch was detected?
As we tag the protocol with finished i would say yes to be consistent.

Reply:
> In "my version" I moved it to a separate folder protocolList, so it would be easier to spot. Since we move it anyway, we might as well do that.

Was that really useful? When i think about how i would check protocol files, it seems more intuitive to have it beside the files.

Reply:
If it's a big list - and it will be since we potentially get the day-delivery für a whole Referat in there - it might not be so easy to spot, being all the way at the bottom.